bin/generate_analytics_JSONL_files.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+jar_dir=$CONDA_PREFIX/share/atlas-cli
+
+scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $scriptDir/common_routines.sh
+
+require_env_var "SOLR_HOST"
+require_env_var "ZK_HOST"
+require_env_var "ZK_PORT"
+require_env_var "BIOENTITIES"
+require_env_var "output_dir"
+require_env_var "EXPERIMENT_FILES"
+require_env_var "jdbc_url"
+require_env_var "jdbc_username"
+require_env_var "jdbc_password"
+require_env_var "server_port"
+
+require_env_var "SPECIES" 
+require_env_var "ACCESSIONS"
+require_env_var "BIN_MAP"
+
+SOLR_PORT=$(get_port_from_hostport $SOLR_HOST)
+SOLR_HOST=$(get_host_from_hostport $SOLR_HOST)
+
+require_env_var "SOLR_PORT"
+
+JAVA_OPTS="-Dsolr.host=$SOLR_HOST"
+JAVA_OPTS="$JAVA_OPTS -Dsolr.port=$SOLR_PORT"
+JAVA_OPTS="$JAVA_OPTS -Dzk.host=$ZK_HOST"
+JAVA_OPTS="$JAVA_OPTS -Dzk.port=$ZK_PORT"
+JAVA_OPTS="$JAVA_OPTS -Ddata.files.location=$BIOENTITIES"
+JAVA_OPTS="$JAVA_OPTS -Dexperiment.files.location=$EXPERIMENT_FILES"
+JAVA_OPTS="$JAVA_OPTS -Djdbc.url=$jdbc_url"
+JAVA_OPTS="$JAVA_OPTS -Djdbc.username=$jdbc_username"
+JAVA_OPTS="$JAVA_OPTS -Djdbc.password=$jdbc_password"
+JAVA_OPTS="$JAVA_OPTS -Dserver.port=$server_port"
+
+# Generate JSONL files from bulk experiments
+
+cmd="java $JAVA_OPTS -jar $jar_dir/atlas-cli-bulk.jar"
```

Comment:
Won't this be redundant now @alfonsomunozpomer ? Also, is JAVA_OPTS taken by all JVMs in general? I remember in the past there were some differences and not all JVMs would recognise it... although maybe that was for other vars.

Reply:
I say redundant since $JAVA_OPTS is normally read directly by the process and we are also passing it here.

Reply:
that's a good point. To make things more confusing, there are other variables: JAVA_TOOL_OPTIONS and _JAVA_OPTIONS